oxd-server/src/test/java/org/gluu/oxd/server/RegisterSiteTest.java

```diff
@@ -160,6 +173,199 @@ public void update(String host) {
+                GrantType.OXAUTH_UMA_TICKET.getValue(),
+                GrantType.CLIENT_CREDENTIALS.getValue()));
+        rp.setFrontChannelLogoutUris(Lists.newArrayList("https://client.example.org/logout"));
+        rp.setTokenEndpointAuthMethod("client_secret_basic");
+        rp.setClientRequestUris(Lists.newArrayList("https://client.example.org/requestUri"));
+        rp.setClientSectorIdentifierUri("https://client.example.org/identifierUri");
+        rp.setRedirectUris(Lists.newArrayList("https://client.example.org/redirectUri"));
+        rp.setRedirectUri("https://client.example.org/redirectUri");
+        rp.setAccessTokenAsJwt(true);
+        rp.setAccessTokenSigningAlg("HS256");
+        rp.setRptAsJwt(true);
+        rp.setResponseTypes(Lists.newArrayList("code"));
+        rp.setAcrValues(Lists.newArrayList("basic"));
+        rp.setContacts(Lists.newArrayList("contact"));
+        rp.setPostLogoutRedirectUris(Lists.newArrayList("https://client.example.org/postLogoutUri"));
+        rp.setScope(Lists.newArrayList("openid"));
+        rp.setLogoUri("https://client.example.org/logoutUri");
+        rp.setClientUri("https://client.example.org/clientUri");
+        rp.setPolicyUri("https://client.example.org/policyUri");
+        rp.setFrontChannelLogoutSessionRequired(true);
+        rp.setTosUri("https://client.example.org/tosUri");
+        rp.setJwks("{\"key1\": \"value1\", \"key2\": \"value2\"}");
+        rp.setIdTokenBindingCnf("4NRB1-0XZABZI9E6-5SM3R");
+        rp.setTlsClientAuthSubjectDn("www.test.com");
+        rp.setSubjectType("pairwise");
+        rp.setRunIntrospectionScriptBeforeAccessTokenAsJwtCreationAndIncludeClaims(true);
+        rp.setIdTokenSignedResponseAlg("HS256");
+        rp.setIdTokenEncryptedResponseAlg("RSA1_5");
+        rp.setIdTokenEncryptedResponseEnc(BlockEncryptionAlgorithm.A128CBC_PLUS_HS256.toString());
+        rp.setUserInfoSignedResponseAlg("HS256");
+        rp.setUserInfoEncryptedResponseAlg("RSA1_5");
+        rp.setUserInfoEncryptedResponseEnc(BlockEncryptionAlgorithm.A128CBC_PLUS_HS256.toString());
+        rp.setRequestObjectSigningAlg("HS256");
+        rp.setRequestObjectEncryptionAlg("RSA1_5");
+        rp.setRequestObjectEncryptionEnc(BlockEncryptionAlgorithm.A128CBC_PLUS_HS256.toString());
+        rp.setDefaultMaxAge(1000);
+        rp.setRequireAuthTime(true);
+        rp.setInitiateLoginUri("https://client.example.org/identifierUri");
+        rp.setAuthorizedOrigins(Lists.newArrayList("https://client.example.org/requestUri"));
+        rp.setAccessTokenLifetime(1000);
+        rp.setSoftwareId("4NRB1-0XZABZI9E6-5SM3R");
+        rp.setSoftwareVersion("2.0");
+        rp.setSoftwareStatement("software name");
+        rp.setClientJwksUri("https://client.example.org/jwksUri");
+        rp.setClaimsRedirectUri(Lists.newArrayList("https://client.example.org/requestUri"));
+
+        return rp;
+    }
+    @Test
+    public void testRegisterRequestMapper() throws IOException {
```

Comment:
This should go into separate RegisterRequestMapperTest.

Reply:
Changes done